packages/bpk-component-nudger/examples.js

```diff
@@ -0,0 +1,203 @@
+        />
+      </div>
+    );
+  }
+}
+
+const options = ['economy', 'premium', 'business', 'first'];
+const compareValues = (a: string, b: string): number => {
+  const [aIndex, bIndex] = [options.indexOf(a), options.indexOf(b)];
+  return aIndex - bIndex;
+};
+
+const incrementValue = (a: string): string => {
+  const currentIndex = options.indexOf(a);
+  const newIndex = currentIndex + 1;
+  if (currentIndex === -1 || newIndex >= options.length) {
+    return a;
+  }
+  return options[newIndex];
+};
+
+const decrementValue = (a: string): string => {
+  const index = options.indexOf(a) - 1;
+  if (index < 0) {
+    return a;
+  }
+  return options[index];
+};
+
+const formatValue = (a: string): string => a.toString();
+
+class ConfigurableNudgerContainer extends Component<{}, { value: string }> {
+  constructor() {
+    super();
+
+    this.state = {
+      value: 'premium',
+    };
+  }
+
+  handleChange = (value: string) => {
+    this.setState({ value });
+  };
+
+  render() {
+    return (
+      <div>
+        <BpkLabel htmlFor="nudger">Traveller Class</BpkLabel>
+        <BpkConfigurableNudger
+          id="nudger"
+          min="economy"
+          max="first"
+          value={this.state.value}
+          onChange={this.handleChange}
+          decreaseButtonLabel="Decrease"
+          increaseButtonLabel="Increase"
+          compareValues={compareValues}
+          incrementValue={incrementValue}
+          decrementValue={decrementValue}
+          formatValue={formatValue}
+          inputClassName={getClassName('bpk-nudger-configurable')}
+        />
+      </div>
+    );
+  }
+}
+
+const DefaultExample = () => (
+  <BpkNudger
+    id="my-nudger"
+    min={1}
+    max={99}
+    value={3}
+    onChange={action('change')}
+    decreaseButtonLabel="Decrease"
+    increaseButtonLabel="Increase"
+  />
+);
+
+const LowerBoundExample = () => (
+  <BpkNudger
+    id="my-nudger"
+    min={3}
+    max={99}
+    value={3}
+    onChange={action('change')}
+    decreaseButtonLabel="Decrease"
+    increaseButtonLabel="Increase"
+  />
+);
+
+const UpperBoundsExample = () => (
+  <BpkNudger
+    id="my-nudger"
+    min={1}
+    max={99}
+    value={99}
+    onChange={action('change')}
+    decreaseButtonLabel="Decrease"
+    increaseButtonLabel="Increase"
+  />
+);
+
+const StatefulExample = () => (
+  <NudgerContainer id="default-nudger" buttonType="secondary" />
+);
+
+const ConfigurableExample = () => <ConfigurableNudgerContainer />;
+
+const OutlineExample = () => (
+  <BpkDarkExampleWrapper>
+    <NudgerContainer id="outline-nudger" buttonType="outline" />
+  </BpkDarkExampleWrapper>
+  // <div className={getClassName('bpk-nudger-outline')}>
+  //   <BpkNudger
+  //     id="outline-nudger"
+  //     min={1}
+  //     max={99}
+  //     value={3}
+  //     onChange={action('change')}
+  //     decreaseButtonLabel="Decrease"
+  //     increaseButtonLabel="Increase"
+  //     buttonType="outline"
+  //   />
+  // </div>
+);
```

Comment:
Looks like this was left in by accident?

Reply:
Yup!